tests/test_genes.py

```diff
@@ -62,3 +64,14 @@ def test_find_fusion_genes(conn, gene):
     result = get_genes_from_variant_types(conn, FUSION_NAMES)
     names = {row['name'] for row in result}
     assert gene in names
+
+
+@pytest.mark.parametrize('gene', CANNONICAL_THERAPY_GENES + CANONICAL_ONCOGENES + CANONICAL_TS)
+def test_get_therapeutic_associated_genes(conn, gene):
```

Comment:
Not a blocker, just a question. Is that calling the test as many time as they are genes in the list? Could we instead, at line 77, have something like 'for gene in genes: assert gene in names' ?

Reply:
I think you are correct and it is simpler, so I've made that change.